python/tank/descriptor/io_descriptor/appstore.py

```diff
@@ -495,16 +440,24 @@ def download_local(self):
 
         # download and unzip
         try:
-            shotgun.download_and_unpack_attachment(sg, attachment_id, target)
+            shotgun.download_and_unpack_attachment(sg, attachment_id, destination_path)
         except ShotgunAttachmentDownloadError, e:
             raise TankAppStoreError(
                 "Failed to download %s. Error: %s" % (self, e)
             )
 
-        # write end receipt
-        filesystem.touch_file(
-            os.path.join(metadata_folder, self._DOWNLOAD_TRANSACTION_COMPLETE_FILE)
-        )
+    def _post_download(self, download_path=None):
+        """
+        code run after the descriptor is successfully downloaded to disk
```

Comment:
nitpick: Please start sentence with capital letter
please document the argument